core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -221,9 +221,9 @@ export class ContributorDashboardPageComponent
 
     this.activeTabName = 'myContributionTab';
 
-    // TODO(#13015): Remove use of unknown as a type.
     this.tabsDetails = {
       ...ContributorDashboardConstants.CONTRIBUTOR_DASHBOARD_TABS_DETAILS
+    // TODO(#13015): Remove use of unknown as a type.
```

Comment:
Actually this is not related to unknown, this can only by solving spread type(strict typing). Add comment so that i remember this.

About issue #13015 is still needed as some unknown come from AngularJs files and some are comes from usage of Jquery. I need to sort out all after GSoC. 

That why i make list in js-ts linter FILES_CONTAIN_UNKNOWN_TYPE.

Reply:
Done, removal of this TODO,

will update #13015 after review of Srijan And Vojta.

Reply:
Done